src/main/java/io/appform/jsonrules/expressions/preoperation/date/DateTimeOperation.java

```diff
@@ -35,12 +35,15 @@ public class DateTimeOperation extends CalendarOperation {
 	protected DateTimeOperation() {
 		super(PreOperationType.date_time);
 	}
-	
-	@Builder
 	public DateTimeOperation(String operand, String zoneOffSet) {
 		super(PreOperationType.date_time, operand, zoneOffSet);
 	}
 
+	@Builder
+	public DateTimeOperation(String operand, String zoneOffSet, String pattern) {
+		super(PreOperationType.date_time, operand, zoneOffSet, pattern);
+	}
+
 	@Override
 	public Number compute(JsonNode evaluatedNode, String operand, String zoneOffset) {
```

Comment:
Remove the implementation, put a delegating implementation in base class ad final and call the new fucntion